Tool 'microsoft.cst.devskim.cli' failed to install

https://github.com/ben-manes/caffeine/actions/runs/3279477252

```
/usr/bin/docker build -t 8d5581:d182c176c6074ef38b83ea8b6dbb4126 -f "/home/runner/work/_actions/microsoft/DevSkim-Action/v1/Dockerfile" "/home/runner/work/_actions/microsoft/DevSkim-Action/v1"
[90](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:90)
  Sending build context to Docker daemon  15.36kB
[91](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:91)
  
[92](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:92)
  Step 1/6 : FROM mcr.microsoft.com/dotnet/sdk:5.0
[93](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:93)
   ---> 9fec788bd1f9
[94](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:94)
  Step 2/6 : RUN mkdir /tools
[95](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:95)
   ---> Using cache
[96](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:96)
   ---> 7e55326a1bf8
[97](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:97)
  Step 3/6 : RUN dotnet tool install --tool-path /tools Microsoft.CST.DevSkim.Cli
[98](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:98)
   ---> Running in 6a075c1ef3ad
[99](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:99)
  error NU1202: Package Microsoft.CST.DevSkim.CLI 0.6.30 is not compatible with net5.0 (.NETCoreApp,Version=v5.0) / any. Package Microsoft.CST.DevSkim.CLI 0.6.30 supports: net6.0 (.NETCoreApp,Version=v6.0) / any
[100](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:100)
  The tool package could not be restored.
[101](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:101)
  Tool 'microsoft.cst.devskim.cli' failed to install. This failure may have been caused by:
[102](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:102)
  
[103](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:103)
  * You are attempting to install a preview release and did not use the --version option to specify the version.
[104](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:104)
  * A package by this name was found, but it was not a .NET tool.
[105](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:105)
  * The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
[106](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:106)
  * You mistyped the name of the tool.
[107](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:107)
  
[108](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:108)
  For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool
[109](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:109)
  The command '/bin/sh -c dotnet tool install --tool-path /tools Microsoft.CST.DevSkim.Cli' returned a non-zero code: 1
[110](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:110)
  
[111](https://github.com/ben-manes/caffeine/actions/runs/3279477252/jobs/5399124373#step:2:112)
Error: Docker build failed with exit code 1

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -1,0 +1,37 @@
+#
+# Test build action
+#
+
+name: Build and test
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ 'main' ]
+  workflow_dispatch:
+
+jobs:
+
+  # Check that container image can be built
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - name: Build action image
+      run: docker build . --file Dockerfile --tag DevSkim-Action:${{ github.sha }}
+
+    # Can the action run successful against the repo
+    - name: Self test action
+      uses: ./
+
+    # Make sure results exist and can by uploaded
+    - name: Upload results
+      uses: github/codeql-action/upload-sarif@v1
+      with:
+        sarif_file: devskim-results.sarif

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0
+FROM mcr.microsoft.com/dotnet/sdk:6.0
 
 RUN mkdir /tools
 

